rabbitmq/datadog_checks/rabbitmq/rabbitmq.py

```diff
@@ -263,7 +268,7 @@ def _get_vhosts(self, instance, base_url, auth=None, ssl_verify=True):
         return vhosts
 
     def check(self, instance):
-        base_url, max_detailed, specified, auth, ssl_verify, custom_tags = self._get_config(instance)
+        base_url, max_detailed, specified, auth, ssl_verify, custom_tags, suppress_warning = self._get_config(instance)
         try:
             vhosts = self._get_vhosts(instance, base_url, auth=auth, ssl_verify=ssl_verify)
```

Comment:
`self._get_vhosts` will also make a request. This line should also be included in the `with` statement below

Reply:
Good catch, thanks for the find!